src/etc/inc/util.inc

```diff
@@ -2536,4 +2536,39 @@ function validateipaddr(&$addr, $type, $label, &$err_msg, $alias=false) {
 
 	return false;
 }
+* returns true if $dhcp6duid is a valid duid entrry */
+function is_duid($dhcp6duid) {
+	$values = explode(":", $dhcp6duid);
+	if (count($values) != 16) {
+		return false;
+	}
+	for ($i = 0; $i < 16; $i++) {
+		if (ctype_xdigit($values[$i]) == false)
+			return false;
+		if (hexdec($values[$i]) < 0 || hexdec($values[$i]) > 255)
+			return false;
+	}
+	return true;
+}
+/* Write the DHCP6 DUID file */
+function write_dhcp6_duid($duidfile) {
```

Comment:
I don't think $duidfile is a good variable name in this case, it's a duid itself and not a path to a file